### PR TITLE
added babyswap-wbnb-busd, activated babyswapfilter

### PR DESCRIPTION
**Pull request by schnarkus**
Output from deployment, all manual tests passed:

Deploying: Moo Baby BNB-BUSD

Vault: 0xBe63A1958dB92D66aB63471e949d059E6A2e47D6
Strategy: 0xfFeb637222c674b1729d2107075886fB9Eb1C02C
Want: 0xDF84C66E5c1E01Dc9CBcbBB09F4cE2A1De6641D6
PoolId: 184

Running post deployment
Setting pendingRewardsFunctionName to 'pendingCake'
Setting call fee to '111'

### CODE REVIEW

File: src/config/vault/bsc.json [hunk 1]
```diff
@@ -59,6 +59,35 @@
...
+      "BATTLE_TESTED",
+      "IL_LOW",
+      "MCAP_LARGE",
+      "AUDIT",
+      "CONTRACTS_VERIFIED"
+    ],
+    "network": "bsc",
+    "addLiquidityUrl": "https://exchange.babyswap.finance/#/add/BNB/0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56",
+    "buyTokenUrl": "https://exchange.babyswap.finance/#/swap?inputCurrency=0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56",
+    "createdAt": 1622825225
```
Comment (sircryptomaxi) [inline src/config/vault/bsc.json:89]: timestamp needs updating to right now, can use https://www.unixtimestamp.com/

**Comment by sircryptomaxi** [comment]
otherwise contracts look good to me

File: src/config/vault/bsc.json (hunk 1 above)
Comment (schnarkus) [reply src/config/vault/bsc.json:89]: okay, do I need another PR for that?

Comment (sircryptomaxi) [reply src/config/vault/bsc.json:89]: no, you can change it in your pull request and push the changes

Comment (schnarkus) [reply src/config/vault/bsc.json]: okay, added the new timestamp 👍

**Pull request closed without merging**